source/quickstart.txt

```diff
@@ -247,6 +247,15 @@ documentation provides details on using the following endpoints:
    * - :ref:`c2c-api-reverse`
      - Reverses the direction of a committed sync operation.
 
+Finalize Cutover Process
+-------------
```

Comment:
We'll want to extend this underline to match the length of the text above it. This currently creates an error in the build log:

```
WARNING(quickstart.txt:250ish): Title underline too short.
```